source/logHandler.py

```diff
@@ -266,14 +281,12 @@ def emit(self, record):
 class FileHandler(logging.FileHandler):
 
 	def handle(self,record):
-		# Only play the error sound if this is a test version.
-		shouldPlayErrorSound =  buildVersion.isTestVersion
 		if record.levelno>=logging.CRITICAL:
 			try:
 				winsound.PlaySound("SystemHand",winsound.SND_ALIAS)

```

Comment:
May be you could fix this to play asynchronously while at it.

```suggestion
				winsound.PlaySound("SystemHand", winsound.SND_ALIAS | winsound.SND_ASYNC)
```